### ISSUE TITLE
Local and S3 Support for RKE backup/restore in the UI

### ISSUE THREAD

**Issue by deniseschannon** [rancher/docs#1028]
These docs should be in the RKE documentation of how to use S3 as well as how to use it in Rancher.

**Comment by MBishop17** [comment]
Need help setting up S3. How to setup etcd backup and restore for RKE docs IN RKE Docs.

There should be another issue for Rancher in the UI.

Amazon Storage. For RKE clusters, we're adding etcd backup for local backups and restore.

**Comment by guangbochen** [comment]
https://github.com/rancher/rke/pull/1036

**Comment by soumyalj** [comment]
@moelsayed : We should include docs to inform users that only after users edit the cluster and "Save" it, the new backup/restore feature in the UI will be available. 
If the users do not edit, they will be using the legacy backups. 

W.r.t snapshot restore, we should include that deleting the dead nodes before performing restore is recommended.
